src/test/java/bio/terra/service/load/LoadDaoUnitTest.java

```diff
@@ -80,8 +87,17 @@ public String getId() {
     }
   }
 
+  private List<Dataset> datasets;
   private List<UUID> loadIdsWithFilesUsedByTest;
 
+  @BeforeAll
+  void beforeAll() throws IOException {
+    datasets =
+        List.of(
+            daoOperations.createDataset(DaoOperations.DATASET_MINIMAL),
+            daoOperations.createDataset(DaoOperations.DATASET_MINIMAL));
```

Comment:
I added a foreign key to our `load` table, so adding new `load` records with dataset IDs will require a corresponding `dataset` record.  That necessitated an expansion to our DB test set-up, but we can set up these records once for the whole class rather than for every test invocation.